cloud/azure/main.tf

```diff
@@ -153,23 +153,23 @@ resource "azurerm_postgresql_server" "civiform" {
   location            = azurerm_resource_group.rg.location
   resource_group_name = azurerm_resource_group.rg.name
 
-  administrator_login          = "psqladmin"
-  administrator_login_password = "H@Sh1CoR3!"
+  administrator_login          = var.postgres_admin_login
+  administrator_login_password = var.postgres_admin_password
 
   // fqdn civiform-db.postgres.database.azure.com
 
-  sku_name   = "GP_Gen5_4"
-  version    = "11"
-  storage_mb = 5120
+  sku_name   = var.postgres_sku_name
+  version    = var.postgres_version
```

Comment:
suggestion(blocking): Upgrading/downgrading postgres version might cause breaking changes in the application so we'd probably want to hard-code this instead of defaulting it in `variables.tf`